storage/ndb/rest-server/data-access-rondb/src/rdrs-hopsworks-dal.cpp

```diff
@@ -348,7 +348,7 @@ RS_Status find_project_team_int(Ndb *ndb_object, HopsworksUsers *users,
     return RS_CLIENT_ERROR("Wrong length of the search key");
   }
 
-  char cmp_str[col_size];
+  char* cmp_str = new char[col_size];
```

Comment:
memory leak. in the schema.go file you will find the table definitions for hopsworks database. create an appropriate constant

Reply:
See above